source/_integrations/zha.markdown

```diff
@@ -64,6 +64,24 @@ There is currently support for the following device types within Home Assistant:
 
 There is also support for grouping of lights, switches, and fans (i.e. support for commanding device groups as entities). At least two entities must be added to a group before the group entity is created. As well as support for binding/unbinding (i.e. bind a remote to a lightbulb or group).
 
+## Introduction and dictionary (terminology vocabulary)
+
+ZHA is a native “Zigbee host application” for Home Assistant that via a Zigbee Coordinator adapter will setup and allow control of a single “Zigbee network” (using mesh topology). Once installed ZHA you will be able to directly join/pair most types of “Zigbee Router Device” and “Zigbee End Device” products to that Zigbee network, regardless of product manufacturer and brand.
+
+Zigbee Router Devices will interconnect with other devices to distribute and repeat transmission messages within the Zigbee network mesh, (they will therefor always need be available so are normally mains-powered).  Zigbee End Devices are usually but not always low-powered products such as battery-operated sensors, these will “sleep” most of the time to save power and only “wake” to report in when changes occur, (example sensor state change like temperature or open and close). 
+
+In most common Zigbee setups you will always need to first setup many Zigbee Router Devices to extend range and coverage of the Zigbee network, otherwise signals and/or messages to far way devices might be lost.
```

Comment:
What does this have to do with an introduction?

Reply:
Maybe more an introduction to Zigbee than to ZHA specifically but I think most/many people do not know this before installing and I think it certainly helps understand that ZHA/Zigbee needs to communicate in a "mesh".

As a backstory, I have seen many new users who right away just ordered a Zigbee Coordinator and loads of battery-operated sensors before installing ZHA then asking in the community why can not connect to sensors far away.

Reply:
> Maybe more an introduction to Zigbee than to ZHA specifically 

This page is not about Zigbee.